resources/views/baskets/index.blade.php

```diff
@@ -4,15 +4,29 @@
 <main>
     <div class="container text-center">
 
-        <h2 class="m-3 inline">My Lists</h2>
+        <h2 class="m-3 inline">My {{ $selectedFilter ?? $selectedFilter}} lists</h2>
+        <div class="container d-flex justify-content-between mb-3">
+            @if($selectedFilter)
+            <a class="py-2 d-md-inline-block text-white filter__link" href="{{ url('baskets') }}">all</a>
+            @endif
+            @foreach($filters as $filter)
+            @if($filter !== $selectedFilter)
+            <a class="py-2  d-md-inline-block text-white filter__link" href="{{ url("baskets/show/{$filter}") }}">{{ $filter }}</a>
+            @endif
+            @endforeach
+        </div>
+
         @foreach ($baskets as $basket)
```

Comment:
Didn't you rename your baskets

Reply:
I did. This commit will be included in the next pull request